### PR TITLE
Newsletter 13: Add RustGameDev Podcast

### PR DESCRIPTION
**Pull request by icefoxen**
Right now it's in the "Learning Material Updates" section because I didn't know where else to put it.  Do we really want it to have its own section?  No idea.

_Part of #232 ("Newsletter 13")_

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -64,6 +64,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+practical questions and issues discovered in the course of creating
+their game, [Nox Futura].  Lots of interesting talk about a new book the
+creator is writing, C++ vs Rust, learning Rust, code architecture and
+ECS's in roguelikes, emergent behavior, and hilarious bugs in Dwarf
+Fortress.
+
+[RustGameDev Podcast Episode 2]: https://rustgamedev.com/episodes/interview-with-herbert-wolverson-bracket-lib
+[Bracket Lib]: https://crates.io/crates/bracket-lib
+[Nox Futura]: https://thebracket.itch.io/nox-futura
+```
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:84]: There's a stray code fence here, think that's the only thing keeping it from going green on CI:
```suggestion
```

**Review by 17cupsofcoffee — COMMENTED** [review]
Content looks good! I agree that having this under learning material seems a bit weird though.

@ozkriff: Would a 'Community Updates' section (or something similar) make sense for things that aren't tied to one project in particular, but to the Rust gamedev community as a whole? Probably wouldn't have stuff to put there every month, though.

**Review by ozkriff — CHANGES_REQUESTED** [review]
It's a nice overview of the second episode, but the first episode was released in August too, so it should be included.

Also, I think it makes sense to announce the podcast as a whole before speaking about individual episodes.

> Do we really want it to have its own section? No idea.

Yeah, this podcast seems like an important community-wide update and I usually put them in top-level sections.

**Comment by ozkriff** [comment]
Oh, and the logo looks like a good fit for the section's image:

image

**Comment by ozkriff** [comment]
Hijacking the PR

**Comment by ozkriff** [comment]
@icefoxen @patchfx r?

**Review by ozkriff — APPROVED** [review]
Hijacked the PR: added some intro and 1st episode.

**Comment by icefoxen** [comment]
Sorry for being out of the loop a bit.  Looks great to me, thanks!

**Review by 17cupsofcoffee — APPROVED** [review]

**Pull request merged**